### PR TITLE
Using install.sh to install the latest version of Easegress

### PR DESCRIPTION
**Pull request by haoel**

### CODE REVIEW

**Review by suchen-sci — APPROVED** [review]
Cool!

**Review by samutamm — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -127,9 +127,7 @@ The basic common usage of Easegress is to quickly set up proxy for the backend s
 We can download the latest or history binaries from the [release page](https://github.com/megaease/easegress/releases). For example, we can install Easegress v1.0.0 for Linux amd64 platform with command:
```
Comment (samutamm) [inline README.md:127]:
```suggestion
We can download the latest or history binaries from the [release page](https://github.com/megaease/easegress/releases). Following shell script will download and extract binaries to `./easegress` folder:
```

**Review by samutamm — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (haoel) [reply README.md:127]: sorry, I merge it too quickly... let me do this in another PR.

**Review by megaeasex — CHANGES_REQUESTED** [review]
[TASK:easegress-pr-test FAILED]megaease/easegress Pull Request 354 Deploy Test failed

**Pull request merged**